.gitignore

```diff
@@ -10,6 +10,7 @@
 .env
 .env.backup
 .env.production
+.php-cs-fixer.cache
```

Comment:
I deleted the accidentally checked in `.php-cs-fixer.cache` and added it to the `.gitignore`.